xml/deployment_hwrecommend.xml

```diff
@@ -23,6 +23,21 @@
   basis. If several processes are located on the same machine, the CPU, RAM,
   disk and network requirements need to be added up.
  </para>
+
+ <sect1 xml:id="multi-architecture">
+  <title>Multiple Architecture Confiurations</title>
+  <para>
+    &productname; supports both &x86; and &amd64; architectures.
+    When considering each architecture, it is important to note that
+    from a cores per OSD, frequency, and RAM perspective, there is no real
+    difference between CPU architectures for sizing.
+  </para>
+  <para>
+    However, it is worth noting that erasure code performance on a lower
+    powered &amd64; CPU could be less performant than on an
+    Intel Xeon or higher-end &amd64; CPU.</para>
+</sect1>
```

Comment:
NOTE: Like with smaller X86_64 processors (non-server), lower-performance ARM cores may not provide an optimal experience, especially when used for erasure coded pools.